core/src/main/java/org/testcontainers/containers/SelinuxContext.java

```diff
@@ -0,0 +1,17 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.model.SELContext;
+import lombok.AllArgsConstructor;
+
+/**
+ * Possible contexts for use with SELinux
+ */
+@AllArgsConstructor
+public enum SelinuxContext {
+    SHARED(SELContext.shared),
+    SINGLE(SELContext.single),
+    NONE(SELContext.none);
+
+    public final SELContext selContext;
```

Comment:
how about private modifier and `@Getter` on it? We usually use getters instead of fields access :)